data-node/src/main/java/org/graylog/datanode/bootstrap/DatanodeBootstrap.java

```diff
@@ -180,24 +94,13 @@ public void configure(Binder binder) {
                 });
     }
 
-    private void setNettyNativeDefaults(Configuration configuration) {
-        // Give netty a better spot than /tmp to unpack its tcnative libraries
-        if (System.getProperty("io.netty.native.workdir") == null) {
-            System.setProperty("io.netty.native.workdir", configuration.getNativeLibDir().toAbsolutePath().toString());
-        }
-        // Don't delete the native lib after unpacking, as this confuses needrestart(1) on some distributions
-        if (System.getProperty("io.netty.native.deleteLibAfterLoading") == null) {
-            System.setProperty("io.netty.native.deleteLibAfterLoading", "false");
-        }
-    }
-
     @Override
     protected void startCommand() {
         final String systemInformation = Tools.getSystemInformation();
 
         final OS os = OS.getOs();
 
-        LOG.info("Graylog {} {} starting up", commandName, version);
+        LOG.info("Graylog Data Node {} {} starting up", commandName, version);
```

Comment:
The log message shows redundant information:

```
[DatanodeBootstrap] Graylog Data Node datanode 6.2.0-SNAPSHOT starting up
```

How about explicitly describing the command? Or we leave the command out.

```suggestion
        LOG.info("Graylog Data Node {} starting up (command: {})", version, commandName);
```